pkg/api/config.go

```diff
@@ -106,6 +106,18 @@ func validateTestStepConfiguration(fieldRoot string, input []TestStepConfigurati
 			validationErrors = append(validationErrors, fmt.Errorf("%s[%d].commands: is required", fieldRoot, num))
 		}
 
+		if len(test.Secret.Name) > 0 {
+			if ok := regexp.MustCompile("^[a-zA-Z]*").MatchString(test.Secret.Name); !ok {
+				validationErrors = append(validationErrors, fmt.Errorf("%s[%d].name: '%s' secret name is not valid value, should be [a-zA-Z]", fieldRoot, num, test.Secret.Name))
```

Comment:
Import the validator from upstream

Reply:
you need to be more specific here. What validator?

Reply:
`k8s.io/kubernetes/pkg/apis/core/validation`

Reply:
the price for this one was quite big. Added missing dependencies in the separate PR for easier review. Not sure if you want to do this for one regexp.

Reply:
hm. I would prefer the regex rather than vendoring the whole library. @petr-muller @stevekuznetsov any thoughts?